auth0/src/main/java/com/auth0/android/request/DefaultClient.kt

```diff
@@ -14,12 +15,24 @@ import java.util.concurrent.TimeUnit
 
 /**
  * Default implementation of a Networking Client.
+ *
+ * @param connectTimeout the connection timeout, in seconds, to use when executing requests. Default is ten seconds.
+ * @param readTimeout the read timeout, in seconds, to use when executing requests. Default is ten seconds.
+ * @param defaultHeaders any headers that should be sent on all requests. If a specific request specifies a header with the same key as any header in the default headers, the header specified on the request will take precedence. Default is an empty map.
```

Comment:
this one was left out of the PR description, can you add it there as well, please?
Also, consider the removal of `setUserAgent` methods in both API clients now that we have these.

Reply:
Added to PR. description. We can discuss removal of setUserAgent seperately